jac-uk/apply
# Feature/1030 Welsh translations of new social mobility questions

## What's included?
Closes #1030 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Apply for a vacancy which is open after 1st April 2023 in Welsh.
2. Go `Equality and diversity` section and check if questions are translated into Welsh.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
